From bba4a9c2bb137794e31fe9fd2a0b079260d76490 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:18 -0400
Subject: [scsi] qla2xxx: Select correct request queue for error type IOCB for ISPFX00

Message-id: <1408391817-6389-31-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87921
O-Subject: [PATCH RHEL 7.1 e-stor 30/69] qla2xxx: Select correct request queue for error type IOCB for ISPFX00.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Saurav Kashyap <saurav.kashyap@qlogic.com>

scsi commit id d550dd273e215040909eb56071d68669e141170f

Signed-off-by: Giridhar Malavali <giridhar.malavali@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_mr.c b/drivers/scsi/qla2xxx/qla_mr.c
index 1f1f112..f3691b8 100644
--- a/drivers/scsi/qla2xxx/qla_mr.c
+++ b/drivers/scsi/qla2xxx/qla_mr.c
@@ -2794,7 +2794,7 @@ qlafx00_error_entry(scsi_qla_host_t *vha, struct rsp_que *rsp,
  srb_t *sp;
  struct qla_hw_data *ha = vha->hw;
  const char func[] = "ERROR-IOCB";
- uint16_t que = MSW(pkt->handle);
+ uint16_t que = 0;
  struct req_que *req = NULL;
  int res = DID_ERROR << 16;
 
-- 
1.7.1